kbs/src/api_server.rs

```diff
@@ -280,8 +283,15 @@ pub(crate) async fn api(
 
                 let claim_str = serde_json::to_string(&claims)?;
 
+                RESOURCE_POLICY_EVALS.inc();
                 // TODO: add policy filter support for other plugins
-                if !core.policy_engine.evaluate(&endpoint, &claim_str).await? {
+                if !core
+                    .policy_engine
+                    .evaluate(&endpoint, &claim_str)
+                    .await
+                    .inspect_err(|_| RESOURCE_POLICY_ERRORS.inc())?
+                {
+                    RESOURCE_POLICY_VIOLATIONS.inc();
```

Comment:
The format of variable `endpoint` looks like `{plugin-name}/{other-parts}`. If you focus on resource related, you can count `endpoint` starts with `resource/` where `{plugin-name}` is `resource`. 

Btw, currently, this policy can indeed cover all plugin calls. The data input to the Policy Engine is a complete `endpoint`.  Thus KBS policy would be better? @fitzthum

Reply:
Okay, thanks for clearing this up!  I think I can just go ahead and do the rename since @fitzthum has already expressed his preference gently but firmly. :-)